### PR TITLE
LinuxCommand: update chmod, find, locate, which, head and tail

### PR DESCRIPTION
**Pull request by Kelly510**
- add basic function of chmod, find, locate, which, head and tail
- add typical examples of chmod, find, locate, which, head and tail

Plan to do: 

- [x] chmod
- [x] find
- [x] locate
- [x] which
- [x] head
- [x] tail

### CODE REVIEW

File: docs/LinuxCommand/chmod.md [hunk 1]
```diff
@@ -71,3 +73,38 @@ s ：特殊权限
 `chmod 751 file`: 给file的属主分配读、写、执行(7)的权限，给file的所在组分配读、执行(5)的权限，给其他用户分配执行(1)的权限
 
 ## 5. 常用形式
+
```
Comment (BobAnkh) [inline docs/LinuxCommand/chmod.md:70]: 注意事项与常用形式有重叠，建议归并

**Review by BobAnkh — CHANGES_REQUESTED** [review]
请修改

**Comment by Kelly510** [comment]
I’ve completed improvement of linux command head and tail, and also modified organization of chmod. Please review @BobAnkh .

File: docs/LinuxCommand/head.md [hunk 2]
```diff
@@ -21,3 +23,15 @@ head [参数] 文件
 默认head命令打印其相应文件的开头10行。可以用负数数字表示除去最后n行。
 
 ## 5. 常用形式
+
+显示file.log文件的开头5行：
```
Comment (BobAnkh) [inline docs/LinuxCommand/head.md:27]: 注意事项里给出了负数数字的说明，你可以把这个移到常用形式作为典型示例

File: docs/LinuxCommand/chmod.md [hunk 3]
```diff
@@ -82,6 +76,18 @@ chmod ugo+r file.txt
...
+```bash
+chmod a-x log2012.log
+```
+
+撤销原来所有的权限，然后使拥有者具有可读权限:
+
+```bash
+chmod u=x log2012.log
+```
+
```
Comment (BobAnkh) [inline docs/LinuxCommand/chmod.md]: 此处有问题，u=x显然不是可读权限

**Review by BobAnkh — CHANGES_REQUESTED** [review]

**Comment by Kelly510** [comment]
I‘ve corrected the two errors. Please review. @BobAnkh

**Review by BobAnkh — APPROVED** [review]

**Pull request merged**